src/server/src/main/java/io/cassandrareaper/core/EditableRepairSchedule.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021-2021 DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cassandrareaper.core;
+
+import io.cassandrareaper.validators.NullOrNotBlank;
+import io.cassandrareaper.validators.ValidEditableRepairSchedule;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.apache.cassandra.repair.RepairParallelism;
+
+@Data
+@NoArgsConstructor
+@ValidEditableRepairSchedule
+public class EditableRepairSchedule {
```

Comment:
We've discussed using Lombok in the early stages of our Reaper fork, but as interesting as it looked we ended up using the Builder pattern as convention for our POJOs and avoiding to use Lombok as it relies on IDE plugins.
While it's fine to reopen that conversation, I think it would be good to be consistent with the existing (and I reckon, undocumented) conventions currently in use.

Reply:
No problem, at this point I don't think I'd worry much about the IDE aspect, it's so popular I believe that all the major IDEs support it well, but I also wasn't sure if there was a process to go through about the addition of a dependency overall and licensing and what not, certainly easier, since the code it replaces is trivial to just take it out for now.

Recent comment removes it, so we can handle it separately if we want it in.

Reply:
sounds good, thanks